### ISSUE TITLE
VSCode opens multiple tabs with the same url

### ISSUE THREAD

**Issue by philolypse** [microsoft/vscode-remote-release#5930]
* Open few windows connected to the remote
* Open and run project that will automatically open browser tab with project url (like [serve](https://www.npmjs.com/package/serve))
* VSCode will open as many tabs as we have windows opened in vscode

**Comment by alexr00** [comment]
> project that will automatically open browser tab

@philolypse what do you expect to cause the browser tabs to open?

**Comment by philolypse** [comment]
My bad, i've provided `serve js` as an example and i don't see an option to automatically open link in browser there. Sorry.

Let's use `xdg-open` command as an example.

> what do you expect to cause the browser tabs to open?

I don't really know how passing the "open link in a browser" event from remote to client process is done, but i suspect that all connected vscode instances are listening for the same thing (which is okay, they should "listen") and that is causing the problem - having multiple tabs opened (in my case) instead of just one.

Another way to test it:
* open multiple windows of vscode
* connect all of them to the same remote
* run e.g. `xdg-open http://localhost` in remote terminal 
* the same amount of browser tabs will be opened as we have connected instances in vscode

I've tested it on:
Client: macOS 12.0.1
Remote: Ubuntu 21.10
VSCode: 1.62.3
